content/docs/command-reference/add.md

```diff
@@ -16,23 +16,30 @@ positional arguments:
 ## Description
 
 The `dvc add` command is analogous to `git add`, in that it makes DVC aware of
-the target data, as a first step to version it. It creates a
+the target data, as a simple step to version it. It creates a
 [DVC-file](/doc/user-guide/dvc-file-format) to track the added data.
 
 The `targets` are files or directories to add with this command, that are turned
 into <abbr>data artifacts</abbr> of the <abbr>project</abbr>. By default, these
 are committed to the <abbr>cache</abbr> (use the `--no-commit` option to avoid
 this, and `dvc commit` to finish the process when needed).
 
-Note that [external data](/doc/user-guide/managing-external-data) (targets
+`dvc add` can be used to version control initial data files and directories from
```

Comment:
Let's simplify it - it can be used to version any file, directory, model that usually large to put into Git. Let's mention pipelines as a separate paragraph if we want to.

Reply:
Done in 22b648c4. Used 2 sentences of a single paragraph though. I think it's easy to read this way.